modules/ossm-remove-operators.adoc

```diff
@@ -6,105 +6,22 @@
 [id="ossm-operatorhub-remove-operators_{context}"]
```

Comment:
This is prolly water under the bridge because this already published, but the guideline is to have the ID match the file name.  See item 2:

https://github.com/openshift/openshift-docs/blob/master/contributing_to_docs/doc_guidelines.adoc#module-file-metadata